src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -94,8 +94,13 @@ class GemUtils {
                 gemsToProcess.each { File gem ->
                     args gem
                 }
+                // there are a few extra args which look like defaults
+                // but we need to make sure any config in $HOME/.gemrc
+                // is overwritten
                 args '--ignore-dependencies',
                      "--install-dir=${destDir.absolutePath}",
+		     '--no-user-install',
+		     '--wrapper',
```

Comment:
yes, wrappers - sorry. and no tests :(

On Wed, Apr 15, 2015 at 10:59 PM, Schalk W. Cronjé <notifications@github.com
> wrote:

> In src/main/groovy/com/github/jrubygradle/GemUtils.groovy
> <https://github.com/jruby-gradle/jruby-gradle-plugin/pull/106#discussion_r28442277>
> :
>
> >                  args '--ignore-dependencies',
> >                       "--install-dir=${destDir.absolutePath}",
> > +		     '--no-user-install',
> > +		     '--wrapper',
>
> Shouldn't that be --wrappers ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jruby-gradle/jruby-gradle-plugin/pull/106/files#r28442277>
> .
>